src/helpers/objectHelper.ts

```diff
@@ -0,0 +1,11 @@
+export const isInt = (value: any) => {
```

Comment:
maybe rename this file: dataHelper as we are dealing with non-object types as well?

Reply:
fixed